index.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
         browser: true,
         es6: true,
         node: true,
+        es2024: true,
```

Comment:
es6 in line 4 and this one are not mutually exclusive?

Reply:
good catch! I think es6 is included in es2024 (because I see it's currently working as es2024), so I'll remove es6.

Reply:
I don't think it makes a difference, the more featured would take over

Reply:
let's remove it to be clear